### PR TITLE
Alot of small fixes and improvements

### LINKED ISSUE #535: Always enable wait command

### LINKED ISSUE #894: Autoheal heals disguised spies
I don't know why, this is just a thing that happens sometimes. Maybe an option to toggle it?

### LINKED ISSUE #939: Add ability to disable automatically setting of PARTY playerlist state
Title. Function that checks this value: IsFriend

### LINKED ISSUE #996: Autobackstab doesn't work if class isn't spy, breaks on randomizer mode
On randomizer mode (and its variants), Auto backstab in triggerbot doesn't function even when knife is out.

### LINKED ISSUE #1117: (Suggestion-Bug) pSilent for projectiles
Fix the pSilent code and perhaps also add a separate GUI checkbox to manually toggle it on and off for preference's sake.

if (projectileAimbotRequired && silent)
    *bSendPackets = false;

### PR DESCRIPTION
**Pull request by BenCat07**
Closes #535, Closes #939, Closes #894, Closes #996, Closes #1117 and a crash in backtrack.

Additionally, some projectile weapons have also been fixed

### CODE REVIEW

File: src/hacks/AutoBackstab.cpp [hunk 1]
```diff
@@ -308,7 +308,7 @@ void CreateMove()
 {
     if (!enabled)
         return;
-    if (CE_BAD(LOCAL_E) || g_pLocalPlayer->life_state || g_pLocalPlayer->clazz != tf_spy || CE_BAD(LOCAL_W) || GetWeaponMode() != weapon_melee || IsPlayerInvisible(LOCAL_E) || CE_BYTE(LOCAL_E, netvar.m_bFeignDeathReady))
+    if (CE_BAD(LOCAL_E) || g_pLocalPlayer->life_state || CE_BAD(LOCAL_W) || GetWeaponMode() != weapon_melee || IsPlayerInvisible(LOCAL_E) || CE_BYTE(LOCAL_E, netvar.m_bFeignDeathReady))
```
Comment (TotallyNotElite) [inline src/hacks/AutoBackstab.cpp:311]: This will make non knife weapons try to backstab. Add a weapon check.

File: src/hacks/MiscAimbot.cpp [hunk 2]
```diff
@@ -20,6 +20,7 @@ static settings::Int sandwichaim_aimkey_mode{ "sandwichaim.aimkey-mode", "0" };
 
 float sandwich_speed = 350.0f;
 float grav           = 0.25f;
```
Comment (TotallyNotElite) [inline src/hacks/MiscAimbot.cpp:22]: Why are these not static constexpr?

File: src/tfmm.cpp [hunk 3]
```diff
@@ -10,6 +10,7 @@
 #include "hacks/AutoJoin.hpp"
 #include "hack.hpp"
 
+static settings::Boolean auto_party{ "player-tools.auto-party", "true" };
```
Comment (TotallyNotElite) [inline src/tfmm.cpp:13]: This is a hilariously bad variable name.

**Review by TotallyNotElite — CHANGES_REQUESTED** [review]

File: src/tfmm.cpp (hunk 3 above)
Comment (aUniqueUser) [reply src/tfmm.cpp:13]: what about player-tools.auto-mark-party ?

Comment (TotallyNotElite) [reply src/tfmm.cpp:13]: or player-tools.set-party-state

Comment (aUniqueUser) [reply src/tfmm.cpp:13]: ye thats better

File: src/hacks/AutoBackstab.cpp (hunk 1 above)
Comment (BenCat07) [reply src/hacks/AutoBackstab.cpp:311]: Did not realize knife check was missing

**Review by TotallyNotElite — APPROVED** [review]
LGTM, gj

**Pull request merged**